src/main/java/com/egg/sp/entities/Message.java

```diff
@@ -1,15 +1,11 @@
 package com.egg.sp.entities;
 
-
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
-
```

Comment:
This space between import groups is important for diferentiating from where a class/annotation/etc was imported, please keep it in your next commits!